packages/app-admin-ui/index.js

```diff
@@ -33,6 +33,12 @@ class AdminUIApp {
       throw new Error('Keystone 5 Admin currently only supports the `PasswordAuthStrategy`');
     }
 
+    if (schemaName === 'internal') {
+      throw new Error(
+        "The schemaName 'internal' is reserved name cannot be used in the AdminUIApp."
```

Comment:
Just a typo, `The schemaName 'internal' is **a** reserved name, **and** cannot be used in the AdminUIApp.`